### DISCUSSION TITLE
Will dpdata support LASP arc-format and MS-car format file?

### DISCUSSION THREAD

**Discussion by QuantumMisaka** [deepmodeling/dpdata#447] [Ideas]
I already have lots of DFT dataset stored by .arc format file. generated by LASP method by using VASP to labeled.
The example of structure file `allstr.arc` and corresponding force and stress file `allfor.arc` are below:

[arcfile_example.tar.gz](https://github.com/deepmodeling/dpdata/files/11184751/arcfile_example.tar.gz)

And I desired to transfer it to `LabeledSystem` format in `dpdata`. and use it in DeePMD
Also, a single `allstr.arc` file can be transfer to `System` format.
which can be easily exported from Materials $tudio , and is also `LASP` output file.

A simple way is to transfer `allstr.arc` and `allfor.arc` data to `OUTCAR` format. and then use `dpdata`
but I think the best way is to add support to this `arc` file in `dpdata`
If supported, it will be an easy way to use "Surface Stochastic Walking" method by LASP to generate dataset for DeePMD.

I already have the will to do it. However, while I'm familiar in treating `allstr.arc` and `allfor.arc` by using single python script, I'm not very familiar with `dpdata`. So I'd like to have some discussion about the support and it's coding, Thanks!

**Comment by njzjz** [comment]
I added documentation for the `Format` class in #451.
